### PR TITLE
Updated ppom and other dependencies

### PR DESCRIPTION
**Pull request by olenagerasimova**
Updated ppom, asto, vertx-server and other dependencies

### CODE REVIEW

File: src/test/java/com/artipie/rpm/http/RpmSliceDownloadITCase.java [hunk 1]
```diff
@@ -29,6 +31,7 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@DisabledOnOs(OS.WINDOWS)
```
Comment (g4s8) [inline src/test/java/com/artipie/rpm/http/RpmSliceDownloadITCase.java:34]: Why it's disabled now?

File: src/test/java/com/artipie/rpm/http/RpmSliceITCase.java [hunk 2]
```diff
@@ -37,6 +39,7 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@DisabledOnOs(OS.WINDOWS)
```
Comment (g4s8) [inline src/test/java/com/artipie/rpm/http/RpmSliceITCase.java:42]: Same question here.

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my questions

File: src/test/java/com/artipie/rpm/http/RpmSliceITCase.java (hunk 2 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/rpm/http/RpmSliceITCase.java:41]: windows github actions do not have docker, that's why these ITs are disabled on win os

**Review by g4s8 — DISMISSED** [review]
@olenagerasimova thanks

**Pull request closed without merging**